test/ci-cd/scripts/comfyui/run-comfyui-tests.sh

```diff
@@ -89,6 +90,9 @@ info "Fetching credentials for project '${cluster_project_id}'"
 ${cluster_credentials_command}
 kubectl get deployment -n ${comfyui_kubernetes_namespace}
 
+step "Wait for ComfyUI deployment to be Available"
+kubectl wait --for=condition=Available deployment/${comfyui_app_name}-${comfyui_accelerator_type} -n ${comfyui_kubernetes_namespace} --timeout=${MAX_WAIT_SECONDS}s
```

Comment:
Did you want to use `${COMFYUI_DEPLOYMENT}` here instead of `${comfyui_app_name}-${comfyui_accelerator_type}`?